account_invoice_base_invoicing_mode/tests/test_base_invoicing_mode.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+
+from odoo.tests.common import SavepointCase
+
+
+class TestBaseInvoicingMode(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
```

Comment:
No tests here, is it normal?

Reply:
There is no test because it is the base module that does not anything without a specific mode.
Maybe could be removed ?

Reply:
It has been removed